resources/analysisTools/qcPipeline/flags_isizes_PEaberrations.pl

```diff
@@ -298,11 +303,22 @@
 		$devsqsum += $howmany * (($i - $average)**2);
 	}
 
-	my $stddev = sqrt($devsqsum/($ctr-1));
-	my $sdrd=int($stddev+0.5);
+	my $stddev;
+	my $sdrd;
+	my $sdperc;
+	my $rounded;
+	if ($ctr - 1 == 0) { # Cannot calculate SD and derived values due to lack of data.
```

Comment:
If you want it a bit shorter, you could assign the "NA" above and invert the if condition. Saves some lines and might be a bit more readable.

Reply:
done